scripts/monitoring/ops-runner.sh

```diff
@@ -31,9 +47,11 @@ if [ $# -eq 0 -o "$1" == "--help" ] ; then
   exit
 fi
 
+FULL_COMMAND="$0 $*"
 NAME=""
 COMMAND=""
 SLEEP=""
+TEMP_FILE=""
```

Comment:
Why does this need to be here?

You set the value a few lines down, and you don't reference it in between the two.